src/screens/WalletScreen.js

```diff
@@ -152,6 +158,21 @@ export default function WalletScreen() {
     }
   }, [assetsSocket, dispatch, fetchedCharts, initialized, sections]);
 
+  useEffect(() => {
+    if (initialized) {
+      InteractionManager.runAfterInteractions(() => {
+        loadGlobalLateData();
```

Comment:
These 3 loads are fully global and do not need to be re-triggered when a user switches wallets